css/selectors/blank.json

```diff
@@ -0,0 +1,60 @@
+{
+  "css": {
+    "selectors": {
+      "empty": {
+        "__compat": {
+          "description": "<code>:blank</code>",
+          "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/:blank",
+          "support": {
+            "webview_android": {
+              "version_added": false
+            },
+            "chrome": {
+              "version_added": false
+            },
+            "chrome_android": {
+              "version_added": false
+            },
+            "edge": {
+              "version_added": false
+            },
+            "edge_mobile": {
+              "version_added": false
+            },
+            "firefox": {
+              "version_added": true,
+              "alternative_name": "-moz-only-whitespace",
```

Comment:
Correction: This should be `:-moz-only-whitespace` (note the `:` at the beginning), all other pseudo‑class selectors in the repository have the leading `:` in the `alternative_name` field.